dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/webdomain/datavalue/DataSetValueQueryParams.java

```diff
@@ -57,7 +55,6 @@
 @AllArgsConstructor( access = AccessLevel.PRIVATE )
 public class DataSetValueQueryParams
 {
-    @NotBlank
```

Comment:
Just checking with @jbee 🤔 No mention of `NotBlank` in https://github.com/dhis2/wow-backend/blob/master/guides/preferred_patterns.md#use-nonnull-in-apis yet 😄 

Do we have a stance on this usage?

Reply:
I think we talked about `@Valid` which would be needed to give `@NotBlank` and other validation annotation an effect in spring layer. There was no agreement to use it.